enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -985,6 +1001,26 @@ def get_group_member_data_with_aggregates(self, request, uuid):
                 enrollment_count = subsidy_learner_aggregate_dict.get(lms_user_id, 0)
             result['enrollment_count'] = enrollment_count
             member_results[key] = result
+
+        # If the request sorts by enrollment count, sort member data and restore the request variables
+        # so we can properly construct the `next` and `previous` pages
+        if sort_by_enrollment_count:
+            member_results.sort(key=lambda result: result.get('enrollment_count'), reverse=(not is_reversed))
+            # Only if the request asked for a specific page of data do we need to restore page and traverse_pagination
+            if request_page:
+                # Needed to construct `next` and `previous` values for the response
+                num_member_results = len(member_results)
+
+                # Cut down the returned "all data" to the page and size requested
+                page_index_start = (request_page - 1) * 10
```

Comment:
Will page size always be 10?  If so, should live in a constant.

Reply:
for the sake of the datatable, yes